src/App.tsx

```diff
@@ -1,10 +1,45 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
 import { USER_ID } from './api/todos';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { ErrorNotifications } from './components/ErrorNotifications';
 
 export const App: React.FC = () => {
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+
+  const [, setLoading] = useState(false);
+
+  const [error, setError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const [inputText, setInputText] = useState('');
+
+  const [selectedFilter, setSelectedFilter] = useState('all');
+
+  useEffect(() => {
+    setLoading(true);
+    client
+      .get<Todo[]>(`/todos?userId=${USER_ID}`)
```

Comment:
Not fixed, use method `getTodos()` here 
<img width="916" alt="image" src="https://github.com/user-attachments/assets/434aee0e-0100-4660-b351-dacbef157362" />